pkg/controller/petset/pet.go

```diff
@@ -63,6 +67,65 @@ type pcb struct {
 	parent *apps.StatefulSet
 }
 
+//Gets the number of replicas stored in pod's StatefulSetReplicasAnnotation. If an the annotation
+//is not present, or if it contains an invalid value, -1 is returned. Otherwise, a 32 bit integer
+//representation of the annotation's value is returned.
+func getReplicas(pod *api.Pod) int32 {
+	if pod == nil {
+		return -1
+	} else if replicaStr, exists := pod.Annotations[StatefulSetReplicasAnnotation]; !exists {
+		glog.Errorf("Pod %s has no annotation for %s", pod.Name, StatefulSetReplicasAnnotation)
+		return -1
+	} else if replicas64, err := strconv.ParseInt(replicaStr, 10, 32); err != nil {
+		glog.Errorf("Pod %s has invalid value %s for annotation %s",
+			pod.Name, replicaStr, StatefulSetReplicasAnnotation)
+		return -1
+	} else {
+		return int32(replicas64)
```

Comment:
I'll change it to replicas, replicas64 was to indicate the return type of the strconv method prior to the explicit int32 cast in the return statement.